pynest/nest/lib/hl_api_connections.py

```diff
@@ -431,9 +410,9 @@ def CGConnect(pre, post, cg, parameter_map=None, model="static_synapse"):
 
     Parameters
     ----------
-    pre : list
+    pre : list or numpy.array
         must contain 1 subnet, or a list of GIDs
-    post : list
+    post : list or numpy.array
         must contain 1 subnet, or a list of GIDs
     cg : connection generator
         libneurosim connection generator to use
```

Comment:
I've updated and extended the documentation in 44e8881 to point out more clearly what `parameter_map` is used for.